feedly/api_client/enterprise/indicators_of_compromise.py

```diff
@@ -0,0 +1,73 @@
+import uuid
+from datetime import datetime
+from typing import Dict, List, Optional
+from urllib.parse import parse_qs
+
+from feedly.api_client.data import Streamable
+from feedly.api_client.session import FeedlySession
+
+
+class FeedlyIoCFetcher:
```

Comment:
I don't think we need to add `Feedly` here

nitpick: I'm not a fan of `Fetcher`, I like `Exporter` or `Downloader` better but maybe that's just me